pkgs/servers/sql/postgresql/ext/vectorchord/package.nix

```diff
@@ -1,68 +1,34 @@
 {
   buildPgrxExtension,
-  cargo-pgrx_0_14_1,
-  clang,
+  cargo-pgrx_0_16_0,
   fetchFromGitHub,
   lib,
   nix-update-script,
   postgresql,
   postgresqlTestExtension,
-  replaceVars,
-  rust-jemalloc-sys,
-  stdenv,
 }:
-let
-  # Follow upstream and use rust-jemalloc-sys on linux aarch64 and x86_64
-  # Additionally, disable init exec TLS, since it causes issues with postgres.
-  # https://github.com/tensorchord/VectorChord/blob/0.4.2/Cargo.toml#L43-L44
-  useSystemJemalloc =
-    stdenv.hostPlatform.isLinux && (stdenv.hostPlatform.isAarch64 || stdenv.hostPlatform.isx86_64);
-  rust-jemalloc-sys' = (
-    rust-jemalloc-sys.override (old: {
-      jemalloc = old.jemalloc.override { disableInitExecTls = true; };
-    })
-  );
-in
 buildPgrxExtension (finalAttrs: {
   inherit postgresql;
-  cargo-pgrx = cargo-pgrx_0_14_1;
```

Comment:
`cargo-pgrx_0_14_1` can now be removed as it should be unused. Any takers?

Reply:
I'll shoot a PR in a bit